recipes/clifford/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "clifford" %}
+{% set version = "0.82" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pygae/{{ name }}/archive/v{{ version }}.zip
+  sha256: ff9ec900c36d28a05307fc259b4465f8c4004e66b6d242f2fb63f4e6d29653b4
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
```

Comment:
Can you try dropping the `build` section of `requirements`? The `build` section I believe is only necessary when you have compiled code for python packages.

Reply:
See: https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L35-L44

Reply:
that looks like it, thanks.